src/game/bettercamera.inc.h

```diff
@@ -264,27 +272,33 @@ static void newcam_rotate_button(void)
     if (newcam_modeflags & NC_FLAG_XTURN)
     {
         if ((gPlayer1Controller->buttonDown & L_CBUTTONS) && newcam_analogue == 0)
-            newcam_yaw_acc = newcam_adjust_value(newcam_yaw_acc,accel);
+            newcam_yaw_acc = newcam_adjust_value(newcam_yaw_acc,accel, 100);
         else if ((gPlayer1Controller->buttonDown & R_CBUTTONS) && newcam_analogue == 0)
-            newcam_yaw_acc = newcam_adjust_value(newcam_yaw_acc,-accel);
+            newcam_yaw_acc = newcam_adjust_value(newcam_yaw_acc,-accel, -100);
```

Comment:
I got `error: too many arguments to function 'newcam_adjust_value'` here

Reply:
Fixed.